src/z3c/checkversions/buildout.txt

```diff
@@ -133,6 +133,9 @@ We are using it directly here:
 True
 >>> p.wait()
 0
+>>> p.stdin.close()
```

Comment:
`Popen.communicate()` reads from, and then closes, its internal file handles. That seems a more common way to address this.

But why deal with it manually at all? The use of a plain `Popen` object could be replaced with `check_output()` which handles the details (internally it uses `communicate()`).

```python
>>> output = subprocess.check_output([sys.executable, main.__file__, '-h'], 
...                                  stderr=subprocess.STDOUT)
>>> output.lower().startswith(b'usage: ')
True
# Nothing to clean up. Raises exception if return code is not 0
```